### PR TITLE
Fix naming and dependencies.

### PR DESCRIPTION
**Pull request by madiator**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,22 +11,23 @@
...
+This validator uses BespokeLabs.AI's minicheck API to evaluate LLM generated text against the provided context. It checks if the claims in the generated text are supported by the given context, using a configurable threshold for determining support. The validator can process the input as a whole or split it into individual sentences for more granular evaluation.
 
 ### Requirements
 
 * Dependencies:
 	- guardrails-ai>=0.4.0
 	- nltk
-	- requests
+	- bespokelabs
+	- tenacity
```
Comment (vutrung96) [inline README.md:22]: why do we need this?

**Review by vutrung96 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (vutrung96) [reply README.md:22]: this == tenacity dependency

Comment (madiator) [reply README.md:22]: It is used to retry etc.
`from tenacity import retry, stop_after_attempt, wait_exponential`

**Review by brianlai98 — COMMENTED** [review]
@madiator can you also change the naming in `tests/test_validator.py` and in `validator.__init__.py`

**Comment by madiator** [comment]
> @madiator can you also change the naming in `tests/test_validator.py` and in `validator.__init__.py`

Ah yes! Sorry about that. Fixed.

**Comment by madiator** [comment]
@aaravnavani I added a comment to your PR. 
I have updated this PR now with the namespace.

**Pull request merged**